### DISCUSSION TITLE
Is readcomic online still supported?

### DISCUSSION THREAD

**Discussion by Tamalero** [Girbons/comics-downloader#109] [Q&A]
I have been trying to latest version ( seem to load fine, no crashes, etc..)
But every url says "issue not found"

Example of Log and command added:

`F:\Download\Windows 10\ComicDownloader>comics-downloader.exe -url=https://readcomiconline.li/Comic/Strange-Academy/Issue-1 -all -format=cbz -debug
time="2022-03-23T23:05:40-06:00" level=info msg=Downloading...
time="2022-03-23T23:05:40-06:00" level=debug msg="Issues Links retrieved: "
time="2022-03-23T23:05:40-06:00" level=error msg="No issues found"`

I understand this project is almost abandoned?

Is there an alternative around?

PS,Thanks to giblongs for all the hard work on this.

**Comment by Girbons** [comment]
Hi @Tamalero,

readcomiconline should be still supported, they might have changed something that does not allow the downloader to retrieve the issues, I will look into it this weekend to try to provide a fix.

> I understand this project is almost abandoned?

I'll always try to provide answer/fixes to make this project up and running don't worry! 😄

**Reply by Tamalero** [reply]
> readcomiconline should be still supported, they might have changed something that does not allow the downloader to retrieve the issues, I will look into it this weekend to try to provide a fix.

Appreciate!

>I'll always try to provide answer/fixes to make this project up and running don't worry! 😄

Gotcha, I was just worried about how quiet the project was XD

**Reply by Girbons** [reply]
Readcomiconline introduced the cloudflare ddos protection, sadly I don't have a workaround for that, sorry
